src/main/kotlin/org/jitsi/nlj/transform/node/Node.kt

```diff
@@ -325,7 +325,7 @@ sealed class StatsKeepingNode(name: String) : Node(name) {
         var maxProcessingDurationNs: Long = -1
     ) {
         private val maxProcessingDurationMs: Double
-            get() = maxProcessingDurationNs / 1000_000.0
```

Comment:
So this was resulting in `-1e-6 ` when `maxProcessingDurationNs` was 0?

Reply:
No, when `maxProcessingDurationNs` is -1 (as initialized)